### PR TITLE
Show playlab walls in a solid color

### PR DESCRIPTION
**Pull request by balderdash**
Per Kiki's request, I'm showing collision mask walls in a solid color for now. It ain't pretty, but it's good enough for levelbuilding purposes until we have proper assets.

image

Starwars obstacle zones and old-style playlab tile obstacles aren't affected, only collision mask walls.

### CODE REVIEW

File: apps/src/studio/collisionMaskWalls.js [hunk 1]
```diff
@@ -100,4 +107,36 @@ export default class CollisionMaskWalls extends Walls {
...
+        const currentByte = wallMap[(y * this.bytesPerRow) + (x / BITS_PER_BYTE)];
+        for (let k = 0; k < BITS_PER_BYTE; k++) {
+          const map = 1 << k;
+          const imageDataIndex = ((y * this.width) + x + k) * BYTES_PER_PIXEL;
+          if (currentByte & map) {
+            // Wall pixel, set color
+            data[imageDataIndex + 0] = color.R;
+            data[imageDataIndex + 1] = color.G;
+            data[imageDataIndex + 2] = color.B;
+            data[imageDataIndex + 3] = 255;
```
Comment (Hamms) [inline apps/src/studio/collisionMaskWalls.js:127]: nit: could more simply do this with `data.splice(imageDataIndex, 4, color.R, color.G, color.B, 255)`

Comment (balderdash) [reply apps/src/studio/collisionMaskWalls.js:127]: Apparently a `Uint8ClampedArray` doesn't know how to `splice`.

**Comment by balderdash** [comment]
Putting it behind an experiment flag is probably a good idea. I used `playlabWallColor`.

File: apps/src/studio/collisionMaskWalls.js (hunk 1 above)
Comment (Hamms) [reply apps/src/studio/collisionMaskWalls.js:127]: Oh, naturally. Ignore this nit, then.

**Pull request merged**